src/SFA.DAS.LevyTransferMatching/Application/Commands/RejectApplication/RejectApplicationsCommand.cs

```diff
@@ -0,0 +1,14 @@
+﻿using MediatR;
+using System.Collections.Generic;
+
+namespace SFA.DAS.LevyTransferMatching.Application.Commands.RejectApplication
+{
+    public class RejectApplicationsCommand : IRequest
+    { 
+        public int PledgeId { get; set; }
+        public int AccountId { get; set; }     
+        public string UserId { get; set; }
+        public string UserDisplayName { get; set; }
+        public List<string> ApplicationsToReject { get; set; }
```

Comment:
As before, I'd have thought a list of integers would be appropriate.

Reply:
I have refactored it and changed it to a list of integers. It makes sense.